addons/html_builder/static/src/plugins/drop_zone_plugin.js

```diff
@@ -126,6 +140,17 @@ export class DropZonePlugin extends Plugin {
         }
 
         this.dropZoneElements = [];
+        this.updateEmptyDropZone();
+    }
+
+    updateEmptyDropZone(force) {
+        if (this.wrapEl.matches(":empty")) {
+            this.emptyDropZone = this.createDropZone();
```

Comment:
More a personal choice, I let you choose what you prefer: I would personally not use `this.emptyDropZone` but rather create a method like `isPageEmpty` or something like this; the goal would be to not store the concept of 'is the page empty' on the `this` but rather to directly check it on the DOM. I think it is less error prone